src/tools/data.js

```diff
@@ -2,9 +2,10 @@
 
 const _ = require('lodash');
 const memoize = require('./memoize');
+const plainModule = require('./plain');
 
 const isPlainObj = (o) => {
```

Comment:
Can we do keep `isPlainObj` and add a `isPlainModule`, and then have the check for `isPlainObj || isPlainModule`? I think that's more clear than making an assumption and changing what `isPlainObj` really means.

Reply:
Looks like `isPlainObj` is used in another `serializeApp` via the `recurseCleanFuncs`. From my understanding, we'll want to permit the `isPlainObj || isPlainModule` check there as well but really I think we _should_ augment `isPlainObj`. The `o.constructor` check is really unforgiving. 

Let me know, if you really think this should be separate and if the recurseCleanFuncs should have the `|| isPlainModule`

Reply:
You have the most context for it, so I'm definitely fine with keeping it like this.